context/app/static/js/components/workspaces/WorkspaceListItem.tsx

```diff
@@ -39,20 +39,15 @@ function WorkspaceListItem({
   toggleItem,
   selected,
   ToggleComponent,
-  disableLaunch = false,
-  disableStop = false,
-  checkMaxDatasets = false,
+  showLaunch = false,
+  showStop = false,
+  disabled = false,
```

Comment:
These default props are now inverted. I imagine you updated all of the existing cases of `WorkspaceListItem`? I see the change for `WorkspacesList`.

Reply:
Yes, that's correct - the other use of that component is in `AddDatasetsFromSearchDialog`, which was updated to reflect the new props.